cli/dependencies.go

```diff
@@ -0,0 +1,130 @@
+package cli
+
+import (
+	"errors"
+
+	"github.com/NethermindEth/sedge/internal/pkg/dependencies"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var deps = []string{
+	dependencies.Docker,
+}
+
+func DependenciesCommand(depsMgr dependencies.DependenciesManager) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deps",
+		Short: "Manage dependencies",
+		Long:  "Checks and install dependencies needed to run Sedge.",
+	}
+	cmd.AddCommand(dependenciesCheckCommand(depsMgr))
+	cmd.AddCommand(dependenciesInstallCommand(depsMgr))
+	return cmd
+}
+
+func dependenciesCheckCommand(depsMgr dependencies.DependenciesManager) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "check",
+		Short: "Check dependencies",
+		Long: `
+Checks if the following dependencies are installed on the host machine:
+	- docker
+	- docker compose
+
+Also checks if the docker engine is running`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			checks := []depCheck{
+				checkDockerIsInstalled,
+				dockerDaemonIsRunning,
+				dockerComposeIsInstalled,
+			}
+			checksOk := true
+			for _, check := range checks {
+				checkOk, err := check(depsMgr)
+				if err != nil {
+					return err
+				}
+				checksOk = checksOk && checkOk
+			}
+			if !checksOk {
+				return ErrMissingDependencies
```

Comment:
Change log message to suggest using `sedge deps install` to install missing dependencies if supported